src/main/java/randoop/main/GenTests.java

```diff
@@ -516,6 +522,71 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
+
+      HashSet<String> flakyTestNames = codeWriter.getFlakyTestNames();
+
+      if (flakyTestNames.size() > 0) {
+        List<ExecutableSequence> regressionSequences = explorer.getRegressionSequences();
+
+        // How many tests an operation occurs in (regardless of how many times it appears in that
+        // test).
+        Map<TypedOperation, Integer> testOccurrences =
+            tallyOperationsInSequences(regressionSequences);
+
+        // TODO: cxing handle Error Test Sequence tallying.
```

Comment:
This TODO should be addressed.

Reply:
At present, error revealing tests aren't reran after they are generated (and thus we can't determine whether they are flaky). This might be a substantial change - should it go in another pull request? I should definitely indicate that in the comment at least.

Reply:
I see.  Just expanding the comment here is adequate.  Thanks!